uniffi_macros/src/export/scaffolding.rs

```diff
@@ -285,7 +285,7 @@ pub(super) fn gen_ffi_function(
             #[doc(hidden)]
             #[no_mangle]
             pub extern "C" fn #ffi_ident(#(#param_names: #param_types,)*) -> ::uniffi::Handle {
-                ::uniffi::deps::log::debug!(#name);
+                ::uniffi::deps::log::trace!(#name);
```

Comment:
I guess I don't really understand why most of the above changes - ie, I'm not sure how exactly the fixture changes really ensures these 2 lines actually work. Or to put it yet another way, I think I'd r+ this with just these 2 lines.

Reply:
I'm not quite sure what you are saying there, but my point is that unless I'm still missing something, I think the test code here is testing the log crate rather than the uniffi crate?

Reply:
The changed code in the test **piggybacks** on an existing test, but it does **indeed** test uniffi crate.

It tests the changed UniFFI source code. Without the changed source code the test would fail. 

The test ensures the level at which UniFFI source code logs name of methods is trace.

Reply:
> Without the changed source code the test would fail.

I hope it's clear that I do infact understand that.

Reply:
@Vinnstah I would benefit from this PR getting landed :D 

Sounds like Mark would prefer if the changes in the test be reverted, maybe you can revert and this might get merged :)

Reply:
To be clear, I'm only weakly against tests for this - IMO the value is lower than the cost. But I'm open to be convinced otherwise, and it's difficult to argue too strongly *against* new tests really :)